<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Gestão Agrícola</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_farms">
       <attribute name="title">
        <string>Fazendas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_add_farm">
            <property name="text">
             <string>Nova Fazenda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh_farms">
            <property name="text">
             <string>Atualizar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_farms">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Localização</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_fields">
       <attribute name="title">
        <string>Talhões</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="combo_farm_fields"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_add_field">
            <property name="text">
             <string>Novo Talhão</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh_fields">
            <property name="text">
             <string>Atualizar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_fields">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Área (ha)</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_crops">
       <attribute name="title">
        <string>Culturas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="combo_field_crops"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_add_crop">
            <property name="text">
             <string>Nova Cultura</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh_crops">
            <property name="text">
             <string>Atualizar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_crops">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tipo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Data Plantio</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_activities">
       <attribute name="title">
        <string>Atividades</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="combo_field_activities"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_add_activity">
            <property name="text">
             <string>Nova Atividade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh_activities">
            <property name="text">
             <string>Atualizar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_activities">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tipo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Descrição</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui> 